src/GiftedChat.tsx

```diff
@@ -53,6 +53,8 @@ import QuickReplies from './QuickReplies'
 export interface GiftedChatProps<TMessage extends IMessage = IMessage> {
   /* Messages to display */
   messages?: TMessage[]
+  /* Typing Indicator state */
+  isTyping?: boolean
```

Comment:
I think it will be interesting to know who is typing, so we may add more the type, maybe:
```ts
isTyping?: boolean | User[]
```

Even if you don't implement it now :)

Reply:
How so? You'll never need to know that you yourself are typing. Unless you're imagining group chat?

Reply:
Yes it's a possibility, but let's keep it as is